mobile/src/main/res/xml/preferences.xml

```diff
@@ -6,61 +6,81 @@
+                android:title="@string/settings_openhab_url"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textNoSuggestions"
+                android:key="default_openhab_username"
+                android:summary="@string/settings_openhab_username_summary"
+                android:title="@string/settings_openhab_username"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textPassword"
+                android:key="default_openhab_password"
+                android:summary="@string/settings_openhab_password_summary"
+                android:title="@string/settings_openhab_password"/>
+        </PreferenceScreen>
+        <PreferenceScreen
+            android:title="@string/settings_openhab_alt_connection">
+            <org.openhab.habdroid.ui.widget.URLInputPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textNoSuggestions"
+                android:key="default_openhab_alturl"
+                android:summary="@string/settings_openhab_alturl_summary"
+                android:title="@string/settings_openhab_alturl"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textNoSuggestions"
+                android:key="default_openhab_remote_username"
+                android:summary="@string/settings_openhab_username_summary"
+                android:title="@string/settings_openhab_username"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textPassword"
+                android:key="default_openhab_remote_password"
+                android:summary="@string/settings_openhab_password_summary"
+                android:title="@string/settings_openhab_password"/>
+        </PreferenceScreen>
+        <PreferenceScreen android:title="@string/settings_openhab_sslsettings">
+            <Preference
+                android:clickable="true"
+                android:defaultValue="@string/settings_openhab_none"
+                android:dependency="default_openhab_demomode"
+                android:key="default_openhab_sslclientcert"
+                android:summary=""
+                android:title="@string/settings_openhab_sslclientcert"/>
+            <Preference
+                android:clickable="true"
+                android:dependency="default_openhab_demomode"
+                android:key="default_openhab_sslclientcert_howto"
```

Comment:
You can't visit it when it's *not* enabled. This indeed works as intended, just tried it, even though I can't tell off-hand *why* it works like that.